design-doc-template.adoc

```diff
@@ -82,73 +94,98 @@ __<List the installation types that are relevant for the features and remove any
 
 == Requirements
 
-__<Describe the requirements that must be fulfilled by this feature.>__
+////
+Describe the requirements that must be fulfilled by this feature.
 
-__<For analyses of a promotion of an existing feature to 'preview' or 'community' stability, only list new requirements;
+For analyses of a promotion of an existing feature to 'preview' or 'community' stability, only list new requirements;
 existing requirements from the feature being promoted are assumed to continue unless otherwise noted in the 'Changed
 requirements' section. Other analyses, including those for promotion to the 'default' stability level, must list all
-requirements.>__
+requirements.
+////
```

Comment:
OK as adoc comment as it reminds the reviewer of how things are meant to be done.